src/MoBi.UI/Extensions/LayoutItemExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DevExpress.LookAndFeel;
+using DevExpress.Skins;
+using DevExpress.XtraLayout;
+
+namespace MoBi.UI.Extensions
+{
+   public static class LayoutItemExtensions
+   {
+      public static void InitializeDisabledColors(this LayoutControl layoutControl)
```

Comment:
@rwmcintosh This is copied from PKSim. We wanted to remove it but in fact we should move it to core so that we can have a disable text that we can actually read